core/src/test/java/nu/yona/server/subscriptions/service/UserServiceTest.java

```diff
@@ -133,20 +127,10 @@ public void postOpenAppEvent_appLastOpenedDateOnSameDay_notUpdated()
 		verify(mockUserRepository, times(0)).save(any(User.class));
 	}
 
-	@Test(expected = InvalidDataException.class)
```

Comment:
Why did you remove this test?

Reply:
Because it's now impossible to create such invalid data: the ``PostPutUser`` does not contain goals anymore.

Reply:
OK